packages.yml

```
@@ -1,12 +1,18 @@
 packages:
+  - git: https://github.com/fivetran/dbt_reddit_ads.git
+    revision: feature/conversion_metrics_by_event
+
+  - git: https://github.com/fivetran/dbt_facebook_ads.git
+    revision: add_custom_action_conversions
+
   - package: fivetran/amazon_ads
     version: [">=1.1.0", "<1.2.0"]
 
   - package: fivetran/apple_search_ads
     version: [">=1.1.0", "<1.2.0"]
 
-  - package: fivetran/facebook_ads
-    version: [">=1.2.0", "<1.3.0"]
+  # - package: fivetran/facebook_ads 
+  #   version: [">=1.3.0", "<1.4.0"]
```

```suggestion
  - package: fivetran/facebook_ads 
    version: [">=1.3.0", "<1.4.0"]
```
After FB Ads is live